core/src/test/java/io/questdb/test/griffin/GroupByTest.java

```diff
@@ -1675,6 +1675,567 @@ public void testSelectMatchingButInDifferentOrderThanGroupBy() throws Exception
+                    null,
+                    true,
+                    true
+            );
+
+            assertPlan(query, "Sort light\n" +
+                    "  keys: [y_utc_15m]\n" +
+                    "    GroupBy vectorized: false\n" +
+                    "      keys: [y_utc_15m]\n" +
+                    "      values: [sum(case([seller='sf',-1.0*volume_mw,buyer='sf',1.0*volume_mw,0.0]))]\n" +
+                    "        SelectedRecord\n" +
+                    "            Async JIT Filter workers: 1\n" +
+                    "              filter: (seller='sf' or buyer='sf')\n" +
+                    "                DataFrame\n" +
+                    "                    Row forward scan\n" +
+                    "                    Frame forward scan on: trades\n");
+        });
+
+
+    }
+
+    @Test
+    public void testLiftAliasesFromInnerSelect5() throws Exception {
+        // Test aliasing a function name
+        assertMemoryLeak(() -> {
+            ddl("create table x ( a int, b int, c symbol, ts timestamp ) timestamp(ts) partition by DAY WAL;");
+            insert("insert into x values (1,2,'3', now()), (2,3, '3', now()), (5,6,'4', now())");
+            insert("insert into x values (1, 5, '4', now()), (1, 3, '1', now())");
+            drainWalQueue();
+            String query =
+                    "select a, sum(b) sum, c as z, count(*) views\n" +
+                            "from x\n" +
+                            "where a = 1\n" +
+                            "group by a,b,z\n" +
+                            "order by a,b,z\n";
+            assertPlan(query,"SelectedRecord\n" +
+                    "    Sort light\n" +
+                    "      keys: [a, b, z]\n" +
+                    "        VirtualRecord\n" +
+                    "          functions: [a,sum,z,views,b]\n" +
+                    "            GroupBy vectorized: false\n" +
+                    "              keys: [a,z,b]\n" +
+                    "              values: [sum(b),count(*)]\n" +
+                    "                SelectedRecord\n" +
+                    "                    Async JIT Filter workers: 1\n" +
+                    "                      filter: a=1\n" +
+                    "                        DataFrame\n" +
+                    "                            Row forward scan\n" +
+                    "                            Frame forward scan on: x\n");
+            assertQuery("" +
+                            "a\tsum\tz\tviews\n" +
+                            "1\t2\t3\t1\n" +
+                            "1\t3\t1\t1\n" +
+                            "1\t5\t4\t1\n",
+                    query,
+                    null,
+                    true,
+                    true);
+        });
+    }
+
+    @Test
+    public void testLiftAliasesFromInnerSelect6() throws Exception {
+        // Test clickhouse plan
+        assertMemoryLeak(() -> {
+            ddl("CREATE TABLE hits\n" +
```

Comment:
nit: DDL can be reduced in size significantly by keeping only the required columns.